coalib/bearlib/abstractions/Linter.py

```diff
@@ -0,0 +1,673 @@
+from contextlib import contextmanager
+import inspect
+from itertools import compress
+import re
+import shutil
+from subprocess import check_call, CalledProcessError, DEVNULL
+from types import MappingProxyType
+
+from coalib.bears.LocalBear import LocalBear
+from coalib.misc.ContextManagers import make_temp
+from coalib.misc.Decorators import assert_right_type, enforce_signature
+from coalib.misc.Future import partialmethod
+from coalib.misc.Shell import run_shell_command
+from coalib.results.Diff import Diff
+from coalib.results.Result import Result
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from coalib.settings.FunctionMetadata import FunctionMetadata
+
+
+def _prepare_options(options):
+    """
+    Prepares options for ``Linter`` for a given options dict in-place.
+
+    :param options:
+        The options dict that contains user/developer inputs.
+    """
```

Comment:
/me sees RadonBear raising a huge issue for this function as soon as we activate that one...